go/src/socialapi/workers/countly/api/api.go

```diff
@@ -0,0 +1,123 @@
+package api
+
+import (
+	"koding/db/mongodb/modelhelper"
+	"koding/tools/utils"
+	"net/http"
+	"net/url"
+	"socialapi/config"
+	"socialapi/models"
+	"socialapi/workers/common/response"
+	"socialapi/workers/countly/client"
```

Comment:
Not sure if tunning heuristics could help or changing the layout of Go code within koding.

I assume there's something that groups imports that do not begin with a known remote repo domain together in the first group. Since we're setting GOPATH inside koding repo, our import paths for local packages are not go-gettable.

E.g. if you instruct goimports explicitely what prefix do your local packages have, then the imports are grouped _almost_ correctly: 

```diff
$ goimports -local socialapi -w api.go
$ git diff
diff --git a/go/src/socialapi/workers/countly/api/api.go b/go/src/socialapi/workers/countly/api/api.go
index ed03ffd110..10ca911a78 100644
--- a/go/src/socialapi/workers/countly/api/api.go
+++ b/go/src/socialapi/workers/countly/api/api.go
@@ -1,10 +1,12 @@
 package api

 import (
-       "koding/db/mongodb/modelhelper"
-       "koding/tools/utils"
        "net/http"
        "net/url"
+
+       "koding/db/mongodb/modelhelper"
+       "koding/tools/utils"
+
        "socialapi/config"
        "socialapi/models"
        "socialapi/workers/common/response"
```